src/protocol.h

```diff
@@ -419,6 +419,8 @@ enum GetDataMsg : uint32_t {
 /** inv message data */
 class CInv
 {
+private:
+    uint32_t type;
```

Comment:
nit: maybe also rename it to `m_type`

Reply:
It is indeed the preferred style for new code, but here it seems like a noisy change for little gain that would make the methods in the class longer and less readable. Holding off for now unless other reviewers think it's worth changing.